packaging/pep517_backend/_backend.py

```diff
@@ -145,14 +151,58 @@ def build_sdist(  # noqa: WPS210, WPS430
             )
         )
 
+        tmp_sdist_directory = tmp_dir / "sdist"
+
         built_sdist_basename = _setuptools_build_sdist(
-            sdist_directory=sdist_directory,
+            sdist_directory=tmp_sdist_directory,
```

Comment:
This feels weird — why is the `sdist_directory` disregarded? I'll probably need to check from the laptop. So far, looking from the GH app gives an impression that things might not be right...